patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -154,12 +162,7 @@
 +        return true;
 +    }
 +
-+    /**
-+     * Called when a user uses the creative pick block button on this entity.
-+     *
-+     * @param target The full target the player is looking at
-+     * @return A ItemStack to add to the player's inventory, empty ItemStack if nothing should be added.
-+     */
++    @Deprecated // Use player sensitive version below.
```

Comment:
Add a `@deprecated use {@link #getPickedResult(RayTraceResult, EntityPlayer)}` javadoc tag here.
If you think this method can be removed in future, add a `// TODO: remove` comment here as well.